packages/avatar/Avatar.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Skeleton from 'react-loading-skeleton';
+import Img from 'react-image';
+import { useEffectAsync } from '@availity/hooks';
+import { avSettingsApi } from '@availity/api-axios';
+import get from 'lodash.get';
+
+const skeletonPropType = PropTypes.shape({
+  width: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  height: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+});
+
+const Loader = ({ skeletonProps, ...rest }) => (
+  <span {...rest}>
+    <Skeleton {...skeletonProps} />
+  </span>
+);
+
+Loader.propTypes = {
+  skeletonProps: skeletonPropType,
+};
+
+Loader.defaultProps = {
+  skeletonProps: {
+    height: '100%',
+  },
+};
+
+const Avatar = ({ fallback, skeletonProps, ...props }) => {
+  const [avatar, setAvatar] = useState(null);
+  const [loading, setLoading] = useState(true);
+
+  let url = avatar;
+
+  useEffectAsync(async () => {
+    setLoading(true);
+    const resp = await avSettingsApi.getApplication('AVATAR');
+
+    const avi = get(resp, 'data.settings[0].avatar');
+
+    setAvatar(avi);
+    setLoading(false);
+  }, []);
+
+  if (!url && loading) {
+    return (
+      <Loader
+        data-testid="avatar-img-loader"
+        skeletonProps={skeletonProps}
+        {...props}
+      />
+    );
+  }
+
+  if (!url && !loading && fallback) {
```

Comment:
Looks like you can just add the `fallback` as the default for the `lodash.get` when it fetches the avatar.

Reply:
hmm... what about on first render before `avSettingsApi.getApplication` resolves?

Reply:
You already have the initial `loading` set to `true` so just set the `initialAvatar` to the fallback.

Reply:
ec9883842bfdb01dc396cae2bde3c9d2f36dc4bb

Reply:
You can just put the fallback here as the initial value of the avatar, and then also in the `get` call you can set the default if it doesn't get it.
https://github.com/Availity/availity-react/commit/ec9883842bfdb01dc396cae2bde3c9d2f36dc4bb#diff-ab6fd2a8dae5b527e60a017e05667e9eR31

Reply:
39c28b4297891be663a925bf79c2c0c8456d3e4b